picard/script.py

```diff
@@ -73,6 +74,11 @@ class ScriptUnknownFunction(ScriptError):
     pass
 
 
+class ScriptRuntimeError(ScriptError):
+    def __init__(self, message, name, line, position):
+        super().__init__("${1:s}:{2:d}:{3:d}: {0:s}".format(message, name, line, position))
```

Comment:
`line` & `position` parameters are in the wrong order.

https://github.com/metabrainz/picard/pull/1508/files#diff-49991922d5e1bb94a02cf28bf4619a1eR1280-R1281